packages/terra-functional-testing/lib/services/wdio-docker-service.js

```diff
@@ -0,0 +1,167 @@
+/* eslint-disable class-methods-use-this */
```

Comment:
Looks like most of these methods are using this. Can we add to the ones that need to be disabled ?

Reply:
We can discuss this a little more, normally I'd agree with this change, but because of how the hooks work in these services they need to be class functions regardless of the presence of `this`. Ideally any method that doesn't use `this` would be static, but the service hooks need to be instances on the class. My preference right now is to leave this disabled for the file, specifically for services.

Reply:
Isn't `initializeSwarm` the only method that isn't using this?

Reply:
`initializeSwam` and `pollCommand` are the two functions not using `this`. I refactored the other functions to use smaller utility functions that worked out to satisfy the `this` lint rule. If we want to remove this eslint-disable I'll likely convert `initializeSwam` and `pollCommand` to static functions rather than ignore them individually. Only the hook methods are problematic with this rule, since they are required to be class methods regardless of `this` being used

Reply:
IMO, this is fine to disable at the top of the file. It'll cover any new methods we add in the future as well or refactoring of functions that may no longer use this.